libraries/botbuilder-ai/botbuilder/ai/qna/models/query_results.py

```diff
@@ -9,11 +9,22 @@
 class QueryResults:
     """ Contains answers for a user query. """
 
-    def __init__(self, answers: List[QueryResult]):
+    _attribute_map = {
```

Comment:
You only need to add the attribute map if you're gonna inherit from msrest.serialization.Model which I suspect you don't need.

In the rest of the sdk we sometimes inherit from msrest because of the convenience that methods to convert from json provide. If you don't do that with this model consider removing it.

Reply:
We would need attrubute map to format key and attribute values. And we are reading these values from rest apis

Reply:
I see, I guess the fix here was to inherit from msrest Model